IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -100,7 +100,10 @@ - (void)handleWebMSAuthResponse:(MSIDWebMSAuthResponse *)response completion:(MS
 
     if (![NSString msidIsStringNilOrBlank:response.upn])
     {
-        NSDictionary *additionalInfo = @{MSIDUserDisplayableIdkey : response.upn};
+        NSMutableDictionary *additionalInfo = [NSMutableDictionary new];
+        additionalInfo[MSIDUserDisplayableIdkey] = response.upn;
+        additionalInfo[MSIDClientInfoKey] = response.clientInfo;
```

Comment:
do we need the whole client info, or we can just pass home account id for which we already have the key? MSIDHomeAccountIdkey

Reply:
changed to MSIDHomeAccountIdkey